devel/qbs/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           qmake5 1.0
 PortGroup           compiler_blacklist_versions 1.0
 
 name                qbs
-version             1.18.2
+version             1.19.1
 revision            1
```

Comment:
when updating to a new version the "revision" field should reset to zero

Reply:
done